src/jit/codegencommon.cpp

```diff
@@ -4850,7 +4850,7 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
             assert(!regArgTab[argNum].processed);
             regArgTab[argNum].processed = true;
             regArgMaskLive &= ~genRegMask(regNum);
-#ifdef FEATURE_MULTIREG_STRUCTS
+#ifdef FEATURE_MULTIREG_ARGS
```

Comment:
It looks like some places use #if and some use #ifdef -- that's bad. It should always be #if, since presumably it's defined 0 or 1, which means #ifdef is always true.